lib/index.js

```diff
@@ -1,6 +1,6 @@
-var addon = require('../native');
+const addon = require('../native');
 
-addon.fibonacci(1000000, (err, result) => {
+addon.fibonacci(100000, (err, result) => {
```

Comment:
I made this change because the `1000000`th fib took too long to compute on my machine. I'm sure this will be the case for others as well